Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -343,6 +343,20 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
           "res/actions/music.png")
       .AddCodeOnlyParameter("currentScene", "")
       .MarkAsComplex();
+  extension
+      .AddAction(
+          "FadeVolume",
+          _("fade the volume of a sound played on a channel."),
+          _("Fade the volume of a sound played on a channel within a given time is seconds."),
```

Comment:
This is not correct English. You don't need to precise the unit here, it's already implicit in the parameter.